packages/fuels/tests/harness.rs

```diff
@@ -2160,13 +2161,21 @@ async fn contract_method_call_respects_maturity() -> anyhow::Result<()> {
 }
 
 #[tokio::test]
-async fn contract_deployment_respects_maturity() -> anyhow::Result<()> {
+#[cfg(feature = "fuel-core-lib")]
```

Comment:
Why is this test limited to only run if we're using fuel core as a lib?

AFAIR `fuel-core` has had a patch release that includes the `fuel-vm` fixes that are needed for these tests to pass. So, as long as we're using the latest `fuel-core` binary these tests should pass. 

Or am I missing something ?

Reply:
This limitation was introduced by the developer because he couldn't make it work lol because of some weird prelude imports 😁 You're right I should make it work regardless.